scripts/k8s/deploy_kubeflow.sh

```diff
@@ -116,13 +102,13 @@ function install_dependencies() {
   esac
 
   # StorageClasse (for volumes and MySQL DB)
-  kubectl get storageclass 2>&1 | grep "(default)" >/dev/null 2>&1
-  if [ $? -ne 0 ] ; then
-      echo "No storageclass found"
-      echo "To setup the nfs-client-provisioner (preferred), run: ansible-playbook playbooks/k8s-cluster/nfs-client-provisioner.yml"
-      echo "To provision Ceph storage, run: ./scripts/k8s/deploy_rook.sh"
-      exit 1
-  fi
+#  kubectl get storageclass 2>&1 | grep "(default)" >/dev/null 2>&1
```

Comment:
Is this no longer required?

Reply:
I commented this out during debugging, will test again with it restored.